pkg/rpc/server/server.go

```diff
@@ -1130,14 +1128,14 @@ func (s *Server) contractScriptHashFromParam(param *request.Param) (util.Uint160
 	}
 	id, err := strconv.Atoi(nameOrHashOrIndex)
 	if err != nil {
-		return result, response.NewRPCError("Unknown contract", "", err)
+		return result, response.NewRPCError("Unknown contract", err)
 	}
 	if err := checkInt32(id); err != nil {
 		return result, response.WrapErrorWithData(response.ErrInvalidParams, err)
 	}
 	result, err = s.chain.GetContractScriptHash(int32(id))
 	if err != nil {
-		return result, response.NewRPCError("Unknown contract", "", err)
+		return result, response.NewRPCError("Unknown contract", err)
```

Comment:
And in cases like this one. It'll be some `ErrKeyNotFound`.

Reply:
Fixed.